pretix_mollie/views.py

```diff
@@ -412,9 +412,7 @@ def handle_order(payment, mollie_id, retry=True):
             OrderPayment.PAYMENT_STATE_CREATED,
             OrderPayment.PAYMENT_STATE_PENDING,
         ):
-            payment.state = OrderPayment.PAYMENT_STATE_CANCELED
-            payment.save()
-            payment.order.log_action("pretix_mollie.event." + data.get("status"))
+            payment.fail(info=payment.info_data, log_data=data.get("status"))
```

Comment:
We usually always log dictionaries, not strings, so I'd suggest
```suggestion
            payment.fail(log_data={"status": data.get("status")})
```